### ISSUE TITLE
Help menu needs some updating

### ISSUE THREAD

**Issue by Azhrei** [RPTools/maptool#1462]
**Describe the bug**
I just ran a walkthru for a new GM and one of his feedback items is that the Help menu wasn't very helpful.  In particular, the **Documentation** link goes to our main web site — we should have a link to the Wiki directly from the **Help** menu as well.

**MapTool Info**
- Version: 1.5.14
- Install: All

**Comment by Azhrei** [comment]
Oh, and would it be acceptable to add to this issue the sorting of the Window menu?  Due to translations, it would need  to be done at run time though.

**Comment by Merudo** [comment]
There is already a Wiki link - check Macro Scripting.

**Comment by Azhrei** [comment]
But a new user may not want to know about scripting, they just want some reference information.  (This is coming from Joe User, someone who knows little about the tool so far, although he had already watched half a dozen of the T. Chapman YouTube videos.)

**Comment by Phergus** [comment]
Yeah.  The Documentation link doesn't go to documentation and the Macro Scripting link goes to the main wiki page rather than, say, the intro to scripting page.

As the wiki could disappear entirely at any time, linking directly to it is probably a bad idea versus linking to a page on the main site that has its own links to the wiki but can be easily updated as needed.

> Oh, and would it be acceptable to add to this issue the sorting of the Window menu? Due to translations, it would need to be done at run time though.

No, it wouldn't be.  :)  Should be it's own thing.

**Comment by Azhrei** [comment]
> No, it wouldn't be. :) Should be it's own thing.

Darn. 😉😁

I'll do that., then.

**Comment by Azhrei** [comment]
I think it was discussed a long time ago to retrieve Help menu items from the main web site rather than a properties file.  They would be cached between fetches.  This would allow older versions of MT to update their menu when locations change without requiring users to download an update.  (Which could be important for campaign continuity.)

It does mean that translations would be more work as a separate file for each language would need to be maintained and the URL would need to be something like http://.../help-menu?lang=<locale> (or perhaps the entire property file could be remote and cached?).

We could point directly to the latest develop commit on GH but if the repo is ever moved, using our own domain and redirecting is safer.

Ideas?
